src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -574,16 +574,24 @@ def process_metric_update_namespace(namespace):
 # region TYPES
 
 
-def datetime_string_type(string):
-    ''' Validates UTC datettime in format '%Y-%m-%d\'T\'%H:%M\'Z\''. '''
-    date_format = '%Y-%m-%dT%H:%MZ'
-    return datetime.strptime(string, date_format).strftime(date_format)
-
-
-def datetime_type(string):
-    ''' Validates UTC datettime in format '%Y-%m-%d\'T\'%H:%M\'Z\''. '''
-    date_format = '%Y-%m-%dT%H:%MZ'
-    return datetime.strptime(string, date_format)
+def get_datetime_type(to_string):
```

Comment:
This should be a keyword argument. So when you call this method, you'd have `get_datetime_type(to_string=False)` etc.

Having `get_datetime_type(False)` looks odd as I can't tell what False means without having to look at the method definition.

Also, then if a better keyword argument that describes things than `to_string` can be thought of, this should be changed. Right now, I don't know what `to_string` actually *means* so I don't have any suggestions for a better replacement. :)